packages/cells/src/cells/date-picker-cell.tsx

```diff
@@ -1,15 +1,87 @@
-import * as React from "react";
-import { CustomCell, CustomRenderer, drawTextCell, GridCellKind } from "@glideapps/glide-data-grid";
+import React from "react";
+import {
+    CustomCell,
+    CustomRenderer,
+    drawTextCell,
+    GridCellKind,
+    ProvideEditorCallback,
+} from "@glideapps/glide-data-grid";
 
 interface DatePickerCellProps {
     readonly kind: "date-picker-cell";
     readonly date: Date | undefined;
     readonly displayDate: string;
-    readonly format: "date" | "datetime-local";
+    readonly format: "date" | "time" | "datetime-local";
```

Comment:
extract to private type

Reply:
Can I rename this to type? I think type is a better variable name as maybe I want to add a way to pass in formatting for a future PR. 

Assuming you're using moment, it would be along the lines of:
`moment(date).format(format)`

Could use any library but just for the current moment's sake... :) 

And for formatting support, I would likely make a PR for that too probably.

Reply:
rename it to `dateKind` as it is the kind of date in the `date` param

Reply:
You will need to support both initially however. We will mark format as deprecated. When you come back and want to add format support we will make the full transition.